modular_splurt/code/modules/jobs/job_types/_job_alt_titles.dm

```diff
@@ -33,7 +33,8 @@
 /datum/job/hos/New()
 	var/list/extra_titles = list(
 		"Security Commander",
-		"Head of Slutcurity"
+		"Head of Slutcurity",
+		"Head of Spookcurity"
```

Comment:
```suggestion
		"Head of Slutcurity"
	)
	if(SSevents.holidays && SSevents.holidays[HALLOWEEN])
		LAZYADD(extra_titles, "Head of Spookcurity")
```
You'll wanna do something like this for each one of the extra titles so they're actually halloween limited